pages/docs/core-concepts/error-handling.mdx

```diff
@@ -25,24 +25,28 @@ Expected errors, also known as _failures_, _typed errors_ or _recoverable errors
 To indicate that our program can fail for some expected reason, we can use the `Effect.fail` constructor:
 
 ```ts twoslash
-import * as Effect from "@effect/io/Effect"
+import * as Effect from "@effect/io/Effect";
```

Comment:
Please remove semi-colons from the code samples - it's just extra syntax noise.

Reply:
Sure, pending resolution to https://github.com/Effect-TS/website/pull/54

Reply:
Will re-format once the build config PR goes in